store/mongo/datastore_mongo.go

```diff
@@ -121,6 +121,9 @@ func NewDataStoreMongo(config DataStoreMongoConfig) (*DataStoreMongo, error) {
 	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
 	defer cancel()
 	c, err := mongo.Connect(ctx, clientOptions)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
can we add Errorf log here?

Reply:
why? how it will be different from what we have right now?

Reply:
Looks okay to me.